src/ResourceManager/Network/Commands.Network/help/Get-AzureRmDdosProtectionPlan.md

```diff
@@ -0,0 +1,116 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+Module Name: AzureRM.Network
+online version:
+schema: 2.0.0
+---
+
+# Get-AzureRmDdosProtectionPlan
+
+## SYNOPSIS
+Gets a DDoS protection plan.
+
+## SYNTAX
+
+```
+Get-AzureRmDdosProtectionPlan [-ResourceGroupName <String>] [-Name <String>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The Get-AzureRmDdosProtectionPlan cmdlet gets a DDoS protection plan.
+
+## EXAMPLES
+
+### Example 1: Get a specific DDoS protection plan
+```
+Get-AzureRmDdosProtectionPlan -ResourceGroupName ResourceGroupName -Name DdosProtectionPlanName
```

Comment:
@murilogr for each of the examples you are providing, please add the corresponding output that the user would see